src/stats/pairhmm/mod.rs

```diff
@@ -102,8 +102,8 @@ pub use pairhmm::PairHMM;
 
 use crate::stats::LogProb;
 
-mod homopolypairhmm;
-mod pairhmm;
+pub mod homopolypairhmm;
+pub mod pairhmm;
```

Comment:
Is this change intended for this pull request? Seems off-topic and if this was not explicitly `pub` before, it should probably stay that way?

Reply:
I was checking the correctness of the output but private modules were not included by default when building the docs, so I made them temporarily public but forgot to restore them. Actually I thought it would be helpful to document items that are currently private. But this should be a separate issue. I've restored them as private modules.